pkg/flow/flow.go

```diff
@@ -245,38 +242,11 @@ func (c *Flow) Run(ctx context.Context) {
 //
 // The controller will only start running components after Load is called once
 // without any configuration errors.
-func (c *Flow) LoadFile(file *File, args map[string]any) error {
+func (c *Flow) LoadFile(file *File, args *map[string]any) error {
```

Comment:
Can we update the PR to not use a pointer to a map? Maps are already fat pointers, so a pointer to a map isn't necessary.